docs/dg/dev/set-up-spryker-locally/redis-configuration.md

```diff
@@ -107,3 +107,74 @@ $config[SessionConstants::ZED_SESSION_PREDIS_CLIENT_OPTION] = [
     'service' => 'mymaster',
 ];
 ```
+
+## Advanced configuration for Redis compression
+
+Standard Redis client configuration uses the environment configuration values set under keys which are defined as constants in `config/Shared/config_default.php`. The standard environment configuration for compression look like this:
+
+By default Redis compression is disabled.
+```php
+$config[RedisConstants::REDIS_COMPRESSION_ENABLED] = getenv('SPRYKER_KEY_VALUE_COMPRESSING_ENABLED') ?: false;
+```
+
+When the compression is activated `SPRYKER_KEY_VALUE_COMPRESSING_ENABLED=true` in environment configuration, the Redis start to compress all data.
+
+The configuration compression settings can be modified and extended at the project level with backward compatibility.
+
+By default, it uses `ZlibCompressorPlugin` to compress/decompress the data.
```

Comment:
please add a note that zlib must be enabled

Reply:
it's enabled by default in core. but can be changed